### PR TITLE
Clean up logic in Backlight widget

### LINKED ISSUE #1866: Missing file for backlight should not crash whole setup
# Issue description

After upgrade qtile 1.5.1 -> 1.6.1, the configuration crashed. The problem is I have same setup for two PC. Both have different GPU: intel and Nvidia. The older version just printed an error message inside the plugin. Not cool, but at least the environment worked. 

I am not sure if it is a bug or a feature, but I prefer to have a more resilient config. I do not see a point of crashing for such situations. 

# Qtile version

1.6.1

# Stack traces
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/libqtile/confreader.py", line 92, in load
    config = __import__(name)  # noqa: F811
  File "/home/neworld/.config/qtile/config.py", line 363, in <module>
    widget.Backlight(
  File "/usr/lib/python3.8/site-packages/libqtile/utils.py", line 213, in class_proxy
    return cls(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/libqtile/widget/backlight.py", line 96, in __init__
    self.max_value = self._load_file(self.max_brightness_file)
  File "/usr/lib/python3.8/site-packages/libqtile/widget/backlight.py", line 105, in _load_file
    raise RuntimeError(
RuntimeError: Unable to read status for max_brightness
```

# Configuration

```
                widget.Backlight(
                    background=color_red,
                    backlight_name = 'intel_backlight',
                ),
```

### PR DESCRIPTION
**Pull request by m-col**
Move button_press logic into mouse_callbacks and fix #1866 and #1868.

### CODE REVIEW

File: libqtile/widget/backlight.py [hunk 1]
```diff
@@ -108,24 +113,22 @@ def _load_file(self, path):
...
     def _get_info(self):
         brightness = self._load_file(self.brightness_file)
-
-        info = {
-            'brightness': brightness,
-            'max': self.max_value,
-        }
-        return info
+        max_value = self._load_file(self.max_brightness_file)
+        return 100 * brightness / max_value
```
Comment (ramnes) [inline libqtile/widget/backlight.py:117]: Should we keep the dict format here?

Comment (m-col) [reply libqtile/widget/backlight.py:117]: What for? _get_info is only used when we want the percentage

Comment (ramnes) [reply libqtile/widget/backlight.py]: Ah sorry, I thought it was related to the IPC.

**Comment by neworld** [comment]
Works fine as before :+1:

**Comment by m-col** [comment]
Looks like something weird happened with the py35 test run. Could someone check it out and see if it was unexpected?

**Comment by ramnes** [comment]
Let's move on and assume the fault is on Travis. :)

**Pull request merged**